### PR TITLE
[Renesas]Modifying some script files

### PR DESCRIPTION
**Pull request by Rbb666**
## 拉取/合并请求描述：(PR description)

[

#### 为什么提交这份PR (why to submit this PR)
修改kconfig,sconscript脚本，添加demo宏控制

#### 在什么测试环境下测试通过 (what is the test environment)
HMI-Board
]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [ ] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [ ] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [ ] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [ ] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [ ] 代码是高质量的 Code in this PR is of high quality
- [ ] 已经使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

**Review by supperthomas — APPROVED** [review]

File: bsp/renesas/ra6m3-hmi-board/board/Kconfig [hunk 1]
```diff
@@ -204,16 +204,61 @@ menu "Hardware Drivers Config"
...
+                bool "Enable LVGL demo for LCD"
+                if BSP_USING_LVGL_DEMO     
+                    choice
+                        prompt "choice lvgl demo"
+                        default BSP_USING_LVGL_MUSIC_DEMO
+
+                        config BSP_USING_LVGL_MUSIC_DEMO
+                            bool "Enable LVGL music demo"
+                            select PKG_USING_LV_MUSIC_DEMO
+                            default y
```
Comment (mysterywolf) [inline bsp/renesas/ra6m3-hmi-board/board/Kconfig:217]: choice语法块没有default y/n 这个表达式，这行可以删掉
choice主要是看 default 或者是 if 以及config的顺序

**Review by mysterywolf — CHANGES_REQUESTED** [review]

File: bsp/renesas/ra6m3-hmi-board/board/Kconfig (hunk 1 above)
Comment (mysterywolf) [reply bsp/renesas/ra6m3-hmi-board/board/Kconfig:217]: 这个地方改了我就没啥问题了

Comment (mysterywolf) [inline bsp/renesas/ra6m3-hmi-board/board/Kconfig:217]:
```suggestion
```

File: bsp/renesas/ra6m3-hmi-board/board/Kconfig [hunk 2]
```diff
@@ -204,16 +204,61 @@ menu "Hardware Drivers Config"
...
+                        config BSP_USING_LVGL_VIDEO_DEMO
+                            bool "Enable LVGL video demo"
+                            select BSP_USING_G2D
+                            select BSP_USING_JPEG
+                            default n
```
Comment (mysterywolf) [inline bsp/renesas/ra6m3-hmi-board/board/Kconfig:222]:
```suggestion
```

**Review by mysterywolf — CHANGES_REQUESTED** [review]

File: bsp/renesas/ra6m3-hmi-board/board/Kconfig [hunk 3]
```diff
@@ -204,16 +204,59 @@ menu "Hardware Drivers Config"
...
+            menuconfig BSP_USING_LVGL_DEMO
+                bool "Enable LVGL demo for LCD"
+                if BSP_USING_LVGL_DEMO     
+                    choice
+                        prompt "choice lvgl demo"
+                        default BSP_USING_LVGL_MUSIC_DEMO
+
+                        config BSP_USING_LVGL_MUSIC_DEMO
+                            bool "Enable LVGL music demo"
+                            select PKG_USING_LV_MUSIC_DEMO
```
Comment (mysterywolf) [inline bsp/renesas/ra6m3-hmi-board/board/Kconfig:216]:
```suggestion
                            select PKG_USING_LV_MUSIC_DEMO

```

**Review by mysterywolf — CHANGES_REQUESTED** [review]

**Review by mysterywolf — APPROVED** [review]

**Pull request merged**